spells/unlock_player.js

```diff
@@ -0,0 +1,35 @@
+// Unlocks a player that is targeted with a "locked spell" fx. annoy
+
+// Takes two params, context = object containing functions to operate on state, payload = data sent from client-side ex username or player-id for the targeted player
+function unlockPlayer(context, payload){
+
+    // First we'll get the game state
+    const game = context.getGameState();
+
+    // In this spell we'll receive the player id (from frontend) (the player to be unlocked)
+    const playerId = payload.player;
+    // We'll extract the player from state.game
+    const player = game.players[playerId];
+
+    // They tried targeting an unknown player
+    if (player === undefined) {
+        context.send('spell:unlock:player:fail', { errorCode: 2 });
```

Comment:
We haven't talked about in what way errorCodes will be used, but for now lets have them start at 0 and go up for each error checking we do. This means this errorCode (since its the first one in your function) should be `0`.